tests/server/test_env_settings.py

```diff
@@ -15,11 +15,21 @@
 
     Contact: code@inmanta.com
 """
+from typing import Any, Dict
 
 from inmanta import data
 from inmanta.util import get_compiler_version
 
 
+def get_environment_setting_default(setting: str) -> Any:
+    return data.Environment._settings[setting].default
+
+
+def check_environment_setting_default(settings_dict: Dict[str, Any]) -> None:
```

Comment:
I find the name of this method rather vague. I would also add a docstring to clarify what this method does.
```suggestion
def check_only_contains_default_setting(settings_dict: Dict[str, Any]) -> None:
```